wp-includes/mysql/class-mysql-token.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+class MySQL_Token {
+	public $type;
+	public $text;
+	public $channel;
```

Comment:
What is the channel actually used for? The variables and methods could use some inline docs in general.

Reply:
@adamziel I got rid of the channel, but will add some TODO comments to consider the token API. Public props here are more performant than getters, so we may want to go with that. Unfortunately, readonly props are not available until PHP 8.1.